ui/components/multichain/import-nfts-modal/import-nfts-modal.js

```diff
@@ -114,7 +114,7 @@ export const ImportNftsModal = ({ onClose }) => {
       nftAddress,
       null,
       tokenId.toString(),
-    );
+    ).catch(() => ({}));
```

Comment:
I guess this call isn't required to pass to verify its a valid nft?  i guess that happens elsewhere, since this one is only used for telemetry purposes.

Reply:
It is not required,  it just keeps the modal in that hanging state until the user manually closes it

Reply:
Would it make sense to at least log something here? Just thinking about how to debug this in the future.

Reply:
Good question; Hmm dnt think its needed for this small case, it should not affect user interactions even if it fails so i guess we can skip adding logs here,